### PR TITLE
Classes - car sale customer

### PR DESCRIPTION
**Pull request by Rahulkug**
## Checklist
- Solution is tested on sphere engine <br/>
    `Note:` Please copy only the solution part. Do not copy/paste the template code. That way, any issues with template code can be caught early on.
    - [ ] Author
    - [x] Reviewer
- PR title is correct - [PR title conventions](https://github.com/DSA-solutions-the10xacademy/JavaScript-DSA-Solutions#title-for-a-pull-request)
    - [ ] Author
    - [x] Reviewer
- Folder/File naming is correct - [Naming conventions](https://github.com/DSA-solutions-the10xacademy/JavaScript-DSA-Solutions#naming-of-files-and-folders)
    - [ ] Author
    - [x] Reviewer
- Coding conventions are followed - [Coding conventions](https://github.com/DSA-solutions-the10xacademy/JavaScript-DSA-Solutions#coding-conventions)
    - [ ] Author
    - [x] Reviewer
- Test cases (action item for **reviewer**) <br/>
    - To be ensured by checking the problem on sphere-engine
    - If any test cases are missing, please add them - [how to add new test cases](https://github.com/DSA-solutions-the10xacademy/JavaScript-DSA-Solutions#adding-new-test-cases)
    - [x] There are at least 2 test cases
    - [x] There is at least 1 test case that is different from the sample test cases

### CODE REVIEW

**Review by rpapavan97 — APPROVED** [review]

File: solutions/classes/car-sale-customer.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+// Your method should be named `getPromisingCustomers`
+// Your method should return the indices of customers who are willing to pay greater than or equal to 90% of the car value
+class CarSell {
+  constructor(customerProposals) {
+    this.customerProposals = customerProposals;
+  }
+  getPromisingCustomers() {
+    let promisingcustomers = [];
+    let count = 0;
+    let flag = false;
```
Comment (sailesh-ramayanam) [inline solutions/classes/car-sale-customer.js:23]: @Rahulkug We don't need `count` and `flag`. If there are no promising cutomers, the array will be of `0` length, in which case you can push `-1`

**Review by sailesh-ramayanam — CHANGES_REQUESTED** [review]

**Comment by sailesh-ramayanam** [comment]
@rpapavan97 
1. First line contains n, an integer denoting the number of customers ~0<n<=100~ (no need to give the bounds for `n`. some students give unnecessary importance to the bounds and get completely carried away from the main point).
2. Output format is described incorrectly - it should say something like `Each line of output should contain the index of a prospective customer. The indices should be output in ascending order. If there is no prospective customer, output -1.`

**Comment by rpapavan97** [comment]
@sailesh-ramayanam , updated the above modifications in problem description

**Comment by prathyusha10x** [comment]
@sailesh-ramayanam @rpapavan97  rahul made the changes in code please review it once he will start the recording

**Review by sailesh-ramayanam — APPROVED** [review]

**Pull request merged**